src/bundle.rs

```diff
@@ -396,6 +396,51 @@ impl<T: Authorization, V> Bundle<T, V> {
             try_note_decryption(&domain, key, action)
         })
     }
+
+    /// Performs trial decryption of each action in the bundle with each of the
+    /// specified outgoing viewing keys, and return the decrypted note contents
+    /// along with the index of the action from which it was derived.
+    pub fn decrypt_outputs_with_ovks(
```

Comment:
I see we have "with_keys" and "for_keys" in here and want to make sure we're using a consistent standard here.

Reply:
I think the "for_keys" naming is a bit unfortunate, and would prefer to rename to 'with_keys' consistently, because since all you need is an IVK, the note may not be "for" you, even though you can see it.

Reply:
"for keys" makes sense when `keys: &[IVK]` because that is the definition of "for": the output has been sent to the key tree containing that IVK, not to "you", so "for you" is not the way to read it. I presume this is what you intended when you added that method @nuttycom: https://github.com/zcash/orchard/commit/e33cd4ade4bad343125347a053f94e4b38bd1f39

That being said, I'm okay to instead follow a consistent `_with_ivks` naming convention here.

I would also prefer that we follow the naming convention from `zcash_note_encryption`, and call this:
```suggestion
    pub fn recover_outputs_with_ovks(
```

Reply:
The protocol spec has "decrypt using ...": https://zips.z.cash/protocol/protocol.pdf#decryptivk
But "decrypt with" is fine.